app/models/scheduled_event.rb

```diff
@@ -59,4 +60,12 @@ def calendar_events(start)
       end
     end
   end
-end
\ No newline at end of file
+
+  def self.day_of_month_for(event_name)
```

Comment:
@nks2109 this is the convenience method I added.

I understand why you're using .where - to avoid the error if nothing is set. But I'd rather catch the error and return nil. This let's us avoid the .trys as well since we validate for start_time.